### PR TITLE
Practice Updates

### PR DESCRIPTION
**Pull request by jschlumkoski**
* Modifying the valence controller logic to incorporate new parameter needed for Insert Element
* Modified urls to point into the new CDN locations

### CODE REVIEW

**Comment by jschlumkoski** [comment]
@grant-cleary Just as a note about swapping the URLS, we have redirects set up on the old paths to the new paths, so anything published with the old URLs will still function!

File: components/plugins/practice.js [hunk 1]
```diff
@@ -514,6 +515,8 @@ class PracticeDialog extends RequesterMixin(Localizer(LitElement)) {
 
 		this._context = this.requestInstance('context');
 
+		console.log(this._context);
```
Comment (grant-cleary) [inline components/plugins/practice.js:518]: I think this made it in by accident.

Comment (grant-cleary) [reply components/plugins/practice.js:518]: Hmm, did you push that change yet? I don't see a new commit.

Comment (jschlumkoski) [reply components/plugins/practice.js]: It's pushed now oops

**Review by grant-cleary — APPROVED** [review]

**Pull request merged**